backends/cadence/fusion_g3/operators/op_rsqrt.cpp

```diff
@@ -27,7 +27,8 @@ namespace native {
 
 namespace {
 
-double rsqrt(double x) {
+template <typename T>
+T rsqrt(T x) {
   return 1.0 / std::sqrt(x);
```

Comment:
think this should be T(1.0) otherwise we're forcing computation in double, right?